execution.bs

```diff
@@ -5011,6 +5032,17 @@ enum class forward_progress_guarantee {
         };
     </pre>
 
+2. Given a subexpression `sndr`, let `Sndr` be `decltype((sndr))`, let `Env` be
+    the type of an environment, and let `rcvr` be a receiver whose associated
+    environment is `Env`. A completion operation is a <dfn
+    export=true>permissible completion</dfn> for `Sndr` and `Env` if its
+    completion signature appears in the argument list of the specialization of
+    `completion_signatures` denoted by `completion_signatures_of_t<Sndr, Env>`.
```

Comment:
Consider moving this definition of /permissible completion/ up to the front-matter to allow it to be used in other sections.